packages/fast-usdc-contract/test/exos/settler.test.ts

```diff
@@ -102,17 +104,33 @@ const makeTestContext = async t => {
     chainId: `${cctpChainInfo.ethereum.namespace}:${cctpChainInfo.ethereum.reference}`,
   });
 
+  const forwardFunds = tx =>
+    flows.forwardFunds(
+      undefined as any,
+      {
+        log, // some tests check the log calls
+        currentChainReference: 'agoric-3',
+        supportsCctp: makeSupportsCctp(chainHub),
+        statusManager,
+        getNobleICA: () => mockAccounts.intermediate.account as any,
+        settlementAccount: Promise.resolve(
+          mockAccounts.settlement.account as any,
+        ),
+      },
+      tx,
+    );
+
```

Comment:
I don't understand why [`orchestrate`](https://github.com/Agoric/agoric-sdk/blob/33eb3c9c00816b734f69a252391c48d461bf622f/packages/orchestration/src/facade.js#L69-L92) as created by `makeOrchestrationFacade` wouldn't work and be better fidelity.

There doesn't seem to be any mock endowments incompatible with async-flow, and it would avoid impedance mismatch between real objects (e.g. statusManager), and the flow's logic.

Reply:
I don't think anyone is claiming it wouldn't work or be higher fidelity. Those aren't the only two criteria.

I'm trying to ship features to users. This is a test. It's testing what I need it to.